internal/controller/policy_subreconciler_webhook.go

```diff
@@ -162,6 +165,9 @@ func (r *policySubReconciler) reconcileMutatingWebhookConfiguration(
 				AdmissionReviewVersions: []string{"v1"},
 			},
 		}
+		if r.featureGateAdmissionWebhookMatchConditions {
+			webhook.Webhooks[0].MatchConditions = policy.GetMatchConditions()
+		}
```

Comment:
Same here, we should inform the user when this propagation cannot be done due to the missing feature

Reply:
done in https://github.com/kubewarden/kubewarden-controller/pull/814/commits/2aa2120f4a30e93fcf9c8300601693b351a01bbe.